dist/routable-component.js

```diff
@@ -381,37 +381,37 @@ var routableComponent =
 	var RoutableComponentRoutes = function () {
 	  /**
 	   * コンストラクタ
-	   * @param {RoutableComponentRouting|RoutableComponentRouting[]} routing_classes ルート定義クラス(の配列)
+	   * @param {RoutableComponentRouting|RoutableComponentRouting[]} routingClasses ルート定義クラス(の配列)
 	   */
 	
 	  function RoutableComponentRoutes() {
-	    var routing_classes = arguments.length <= 0 || arguments[0] === undefined ? [] : arguments[0];
+	    var routingClasses = arguments.length <= 0 || arguments[0] === undefined ? [] : arguments[0];
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Warning - Unexpected var
* Error - Expected indentation of 2 space characters but found 0. (indent)
* Error - Use the rest parameters instead of 'arguments'. (prefer-rest-params)
* Warning - No magic number
* Error - Unexpected use of undefined. (no-undefined)

<!-- Comment by [SideCI](https://sideci.com) -->